example/iga/src/mapping_handler.h

```diff
@@ -38,8 +42,8 @@ class MappingHandler {
     std::array<ParamCoord, DIM> param_coords{};
     for (int i = 0; i < DIM; ++i) {
       iga::elm::Element elm = elm_gen.GetElementList(i)[elm_gen.GetElementIndices(element_number)[i]];
-      param_coords[i] = ParamCoord{((elm.GetUpperBound() - elm.GetLowerBound()).get() * itg_pnts[i]
-          + (elm.GetUpperBound() + elm.GetLowerBound()).get()) / 2.0};
+      param_coords[i] = ParamCoord{((elm.GetUpperBound() - elm.GetLowerBound()).get() * itg_pnts[i] +
+          (elm.GetUpperBound() + elm.GetLowerBound()).get()) / 2.0};
```

Comment:
Please fix alignment of operands (bold): ... {(**(...).get()** ... + **(...).get()**) / 2.0};